example_configs/dropwizard-0-8-x.yml

```diff
@@ -0,0 +1,153 @@
+---
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+blackListObjectNames: ["metrics:type=*,name=jvm*"]
+
+rules:
+
+- pattern: "metrics<name=ch.qos.logback.core.Appender.(debug|error|info|trace|warn)><>Count"
+  name: log_msg_total
```

Comment:
it's probably best to prefix all of these with `logback`. Are you aware that the java client has a logback module?